adala/utils/exceptions.py

```diff
@@ -0,0 +1,6 @@
+
+class ConstrainedGenerationError(Exception):
+    def __init__(self):
+        self.message = "The selected provider model could not generate a properly-formatted response"
```

Comment:
can we just do `message = "..."` at the class level?

Reply:
I believe this is the way to init it, that way does not work 
```
>>> class ConstrainedGenerationError(Exception):
...     message = "hey bud"
... 
>>> e = ConstrainedGenerationError() 
>>> str(e)
''
```